examples/304-log-errors.js

```diff
@@ -0,0 +1,178 @@
+import { check } from 'k6';
+import launcher from 'k6/x/browser';
+
+export const options = {
+  scenarios: {
+    err: {
+      executor: 'shared-iterations',
+      vus: 1,
+      iterations: 20,
+      exec: 'locatorStrictMode',
+      // exec: 'pageGotoTimeout0',
+      // exec: 'pageGotoTimeout100',
+      // exec: 'pageQuery',
+      // exec: 'pageWaitForSelector',
+    }
+  }
+}
+
+export function locatorStrictMode() {
+  const browser = launcher.launch('chromium', {
+    headless: __ENV.XK6_HEADLESS ? true : false,
+  });
+  const context = browser.newContext();
+  const page = context.newPage();
+
+  page.goto('https://test.k6.io/', { waitUntil: 'networkidle' });
+
+// ===== BEFORE =====
+// ERRO[0002] click: element handle cannot evaluate: cannot call function on expression ("\n\t\t(node, injected, selector, strict, state, timeout, ...args) => {\n\t\t\treturn injected.waitForSelector(selector, node, strict, state, 'raf', timeout, ...args);\n\t\t}\n\t\n//# sourceURL=__xk6_browser_evaluation_script__\n") in execution context (3) in frame (F229F0E745BC94879FA7D9EA9AC1CA63) with session (DEE3FBD05BF682C55E68A2D972EA215F): exception "Uncaught (in promise)" (0:0): error:strictmodeviolation
+//         at reflect.methodValueCall (native)
+//         at locatorStrictMode (file:///home/ivan/Projects/grafana/xk6-browser/examples/304-log-errors.js:144:15(41))
+//         at native  executor=shared-iterations scenario=err source=stacktrace
+//
+// ===== AFTER =====
+// TODO: Should we make this error:strictmodeviolation friendlier?
```

Comment:
I remember that there was some confusion as to what strict mode actually was (https://github.com/grafana/xk6-browser/pull/390#discussion_r895472111), so maybe it would be good to clarify that it it found more than one element that matched the selector?